integrations/pgsql.ini.tmpl

```diff
@@ -10,6 +10,9 @@ PASSWD   = {{TEST_PGSQL_PASSWORD}}
 SSL_MODE = disable
 PATH     = data/gitea.db
 
+[indexer]
+ISSUE_INDEXER_PATH = integrations/indexers/issues.bleve
```

Comment:
Do not use same path for mysql, pgsql and sqlite as they will conflict when run in parallel